plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -798,4 +799,26 @@ protected function fetchUrl($url)
     {
         return Http::fetchRemoteFile($url);
     }
+
+    /**
+     * Updates the DB-IP URL option value so that users see
+     * the updated link in the "Download URL" field on the plugin page
+     * instead of the one that was set when Matomo was installed months
+     * or even years ago.
+     *
+     * @param  string  $url The URL retrieved from the options
+     */
+    protected function updateDbIpUrlOption(string $url): void
```

Comment:
Could you maybe change the method, so it's also able to update the ISP database url as well. Someone could for example use the free asn database from DB-IP (https://download.db-ip.com/free/dbip-asn-lite-2020-11.mmdb.gz)

Reply:
Of course - I wasn't sure if the ISP DB was also sources from DB-IP so I limited this method to updating the location URL.

Reply:
Done - I set the ISP DB link to `https://download.db-ip.com/free/dbip-asn-lite-2020-04.mmdb.gz` in my options and backdated the location DB URL as well, and re-run the updater task - it updated both URLs to the November ones.